test/functional/legacy/mapping_spec.lua

```diff
@@ -46,4 +46,29 @@ describe('mapping', function()
       +
       +]])
   end)
+
+  it('i_CTRL-G_U', function()
+    -- <c-g>U<cursor> works only within a single line
+    execute('imapclear')
+    execute('imap ( ()<c-g>U<left>')
+    feed('G2o<esc>ki<cr>Test1: text with a (here some more text<esc>k.')
+    -- test undo
+    feed('G2o<esc>kiTest2: text wit a (here some more text [und undo]<c-g>u<esc>k.u')
+    execute('imapclear')
+    execute('set whichwrap=<,>,[,]')
+    feed('G3o<esc>2k')
+    execute([[:exe ":norm! iTest3: text with a (parenthesis here\<C-G>U\<Right>new line here\<esc>\<up>\<up>."]])
+
+    expect([[
+      
+      
+      Test1: text with a (here some more text)
+      Test1: text with a (here some more text)
+      
+      Test2: text wit a (here some more text [und undo])
+      new line here
+      Test3: text with a (parenthesis here
+      new line here
+      ]])
```

Comment:
Original `test_mapping.ok` contains some extra lines, compared to my migrated `expect()` value above:

```
Test1: text with a (here some more text)
Test1: text with a (here some more text)


Test2: text wit a (here some more text [und undo])

new line here
Test3: text with a (parenthesis here
new line here
```

I can't figure out why.